### DISCUSSION TITLE
API 질문

### DISCUSSION THREAD

**Discussion by ByeongGi** [devmmin/meet-us#39] [Q&A] [answered]
@devmmin 민정님 질문 

1. API 성공과 실패 시 response 구조가 다르던데 이거 혹시 맞춰주실수 있나요?
성공 시 data 가 넘어옮
실패 시 statusCode, message, … 등이 넘어옮

2. API 요청하는 accessToken 만료 시 refreshToken 을 요청값에 담아서 '/v1/auth/refresh' API를 요청하게 되는데
(제가 이해하고 있는거 맞겟죠..?)
API response 에서 accessToken 만료 에러코드는 몇일까요?
(스웨거에서 200, 401만 발견했는데..
혹시 다른곳을 참고해봐야할까요?)
현재 해당 케이스 재현 어떻게 해야하는지 몰라서 발견을 못했는데요,
만약 401이면 다른 인증 실패와 겹칠거 같은데 statusCode 401 아니고 다른 값으로 올까요?

**Comment by ByeongGi** [comment, marked as answer]
1. api 모델 맞추어보도록 하겠습니다. (성공 실패시 일관된 API Response)
2. 액세스 토큰 만료시에 refresh api 를 호출해서 새로운 엑세스 토큰을 받는거 맞음

**Reply by devmmin** [reply]
오와.. 정리해주셔서 감사해요 저번에 메일 왔는데, 이제야 제대로 봤네요!! 캬 감사합니다~!
